src/main/java/com/rtb/projectmanagementtool/pages/HomePageServlet.java

```diff
@@ -0,0 +1,91 @@
+/** Servlet responsible getting data for loading home page */
+package com.rtb.projectmanagementtool.pages;
+
+import com.google.appengine.api.datastore.*;
+import com.google.gson.Gson;
+import com.rtb.projectmanagementtool.auth.*;
+import com.rtb.projectmanagementtool.project.*;
+import com.rtb.projectmanagementtool.task.*;
+import com.rtb.projectmanagementtool.user.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/home")
+public class HomePageServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    // Initialize controllers
+    ProjectController projectController = new ProjectController(datastore);
+    TaskController taskController = new TaskController(datastore);
+    UserController userController = new UserController(datastore);
+    // AuthOps auth = new AuthOps(userController);
+
+    // Get user
+    Long userId = Long.parseLong(request.getParameter("userId"));
+    /**
+     * TODO: change to Long userId = auth.whichUserLoggedIn(request, response); I'm not entirely
+     * sure how authentication works yet
+     */
+    response.setContentType("application/json;");
+
+    // If user is not logged in, return
+    if (userId == -1) {
+      response
+          .getWriter()
+          .println(
+              getJsonForResponse(
+                  /*userLoggedIn*/ false,
+                  /*user*/ null,
+                  /*userProjects*/ null,
+                  /*userTasks*/ null));
+      return;
+    }
```

Comment:
This is a good starting point for the authentication piece.  I would suggest moving this up higher though, so we don't have to initialize the controllers if the user is not logged in anyways. Then maybe adding a "BuildNotLoggedInResponse()" method (which passes the false/null/null/null params to the response builder) just so we don't have to pass the false/null params every time.

@Tzirliya @talgsternberg